pkg/clients/integration/integration_test_scenarios.go

```diff
@@ -62,6 +62,46 @@ func (i *IntegrationController) CreateIntegrationTestScenarioWithEnvironment(app
 	return integrationTestScenario, nil
 }
 
+// CreateIntegrationTestScenario creates beta1 version integrationTestScenario.
+// Universal method to create a IntegrationTestScenario (its) in the kubernetes clusters and adds a suffix to the its name to allow multiple its's with unique names.
+// Generate a random its name with #combinations > 11M, Create unique resource names that adhere to RFC 1123 Label Names
+// https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
+func (i *IntegrationController) CreateIntegrationTestScenarioV2(itsName, applicationName, namespace, gitURL, revision, pathInRepo string) (*integrationv1beta1.IntegrationTestScenario, error) {
```

Comment:
Only difference when compare to `CreateIntegrationTestScenario` is providing ITS name. Could we extend `CreateIntegrationTestScenario` to handle both cases instead of adding new function here?

Reply:
itsName is a new param in CreateIntegrationTestScenarioV2
I shall solve the function duplication in here as well

Reply:
itsName is a new parameter in CreateIntegrationTestScenarioV2 where I need to randomize the its name
CreateIntegrationTestScenario creates the its name as - "my-integration-test-" + util.GenerateRandomString(4)
I'll make sure to preserve this behaviour as a default behaviour for backward compatibility and still avoid duplication

Reply:
Fixed the duplication issue